components/post/post_component.tsx

```diff
@@ -480,7 +481,7 @@ const PostComponent = (props: Props): JSX.Element => {
                 onMouseOver={handleMouseOver}
                 onMouseLeave={handleMouseLeave}
             >
-                {Boolean(isSearchResultItem) &&
+                {(Boolean(isSearchResultItem) || props.location === Locations.SEARCH) &&
```

Comment:
@nevyangelova About this chagne, Asaad told use that we could leave in case of Pinned post https://community.mattermost.com/core/pl/uegre6shktf8zpzzu8zce3hiea

Reply:
cc// @asaadmahmood

Reply:
@AshishDhama I have spinned this test server with 7.1 version and the pinned posts should have the headline too 
https://nevytest.test.mattermost.cloud/lalalalalalal/channels/town-square

Reply:
Yeah check the thread we had that discussion

Reply:
ok I will remove it

Reply:
But I leave that to asaad, and to make it more clear what you have here is exactly like it was before common post component.

Reply:
Still it seems per discussion that this is a redundant information, removing for now as to not block the release fix and we can add it later if we decide :)